quest_test.go

```diff
@@ -409,11 +409,6 @@ func TestSmoke_AllUsersAPI(t *testing.T) {
 	AssertRole(t, NewGlob.QueryClient, "dummyrole", dummyRole)
 
 	CreateUser(t, NewGlob.QueryClient, "dummyuser")
-	AssignRolesToUser(t, NewGlob.QueryClient, "dummyuser", []string{"dummyrole"})
-	AssertUserRole(t, NewGlob.QueryClient, "dummyuser", "dummyrole", dummyRole)
-	RegenPassword(t, NewGlob.QueryClient, "dummyuser")
-	DeleteUser(t, NewGlob.QueryClient, "dummyuser")
-
```

Comment:
why removing all this?

Reply:
CreateUser will fail without the role, hence AssignRolesToUser will not work.
There are two more tests -
1. TestSmoke_NewUserNoRole - that should give 400 status 
2. TestSmokeRbacBasic - that creates user with role which should give 200 status

Reply:
why can't we add the role in the `CreateUser` call?